### DISCUSSION TITLE
What are recommended training step counts when Im training for a lighting style for exteriors?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1045] [Q&A] [answered]
Any other tips to know when training for this kind of style training?
I have about 35 instance images and 250 concept images.

**Comment by TheLastBen** [comment, marked as answer]
that looks right, but you should know that there is no standard setting, it depends on the instance images and the subject trained, so you'll have to experiment, start with a small amount of steps, then add more slowly.

**Reply by gilroff** [reply]
Thanks, I will.
